package.json

```diff
@@ -59,4 +59,4 @@
     "typescript": "^5",
     "vitest": "^1.6.0"
   }
-}
\ No newline at end of file
+}
```

Comment:
This change is not necessary. it's just a blank line. remove it

Reply:
Done @JulianaSau